File: src/PiecykPolHurt/PiecykPolHurt.Model/Queries/ReportQuery.cs
using PiecykPolHurt.Model.Enums;

namespace PiecykPolHurt.Model.Queries
{
    public sealed class ReportQuery : ListQuery
    {
        public string Title { get; set; }
        public string Group { get; set; }
        public ReportSortOption? SortOption { get; set; }
    }
}

File: src/piecyk-pol-hurt-frontend/src/API/Models/Reports/ReportQuery.ts
export interface ReportQuery {
    pageSize: number;
    pageNumber: number;
    title?: string;
    group?: string;
    sortOption?: ReportSortOption;
}

export enum ReportSortOption
{
    TitleAsc = 1,
    TitleDesc = 2,
    GroupAsc = 3,
    GroupDesc = 4
}